gammapy/modeling/models/spectral.py

```diff
@@ -163,6 +163,24 @@ def integral(self, emin, emax, **kwargs):
         else:
             return integrate_spectrum(self, emin, emax, **kwargs)
 
+    def integral_error(self, emin, emax):
+        """Evaluate the error of the integral flux of a given spectrum in
+        a given energy range.
+
+        Parameters
+        ----------
+        emin, emax : `~numpy.array`
+            Arrays of minimum and maximum energies.
+
+        Returns
+        -------
+        flux, flux_err : tuple of `~astropy.units.Quantity`
+            Integral flux and flux error betwen emin and emax.
+        """
+        flux = self.integral(emin,emax)
+        flux_err = flux * self.evaluate_error(emin, epsilon=1e-4)[1] / self(emin)
```

Comment:
Minor: as `evaluate_error()` returns both dude as well as dude_err, there is no need to recompute it. You can just use  `dnde, dnde_err = self.evaluate_error()` and maybe use the center energy like `np.sqrt(e_min * e_max)`

Reply:
Done.